x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/QueryRuleCriteria.java

```diff
@@ -23,18 +27,32 @@
 import org.elasticsearch.xcontent.XContentType;
 
 import java.io.IOException;
+import java.util.List;
 import java.util.Locale;
 import java.util.Objects;
 
 import static org.elasticsearch.xcontent.ConstructingObjectParser.constructorArg;
+import static org.elasticsearch.xcontent.ConstructingObjectParser.optionalConstructorArg;
+import static org.elasticsearch.xpack.application.rules.QueryRuleCriteria.CriteriaType.GLOBAL;
 
 public class QueryRuleCriteria implements Writeable, ToXContentObject {
     private final CriteriaType criteriaType;
     private final String criteriaMetadata;
-    private final Object criteriaValue;
+    private final List<Object> criteriaValues;
+
+    private static final Logger logger = LogManager.getLogger(QueryRuleCriteria.class);
 
     public enum CriteriaType {
-        EXACT;
+        GLOBAL,
+        EXACT,
```

Comment:
Naming: Maybe using MATCH would be more ES-like:
```suggestion
        MATCH,
```

Reply:
That's a good point. @aznick - What do you think of the naming suggestions in this review? 
- Exact match `EXACT` --> `MATCH`
- Fuzzy match `EXACT_FUZZY` --> `FUZZY_MATCH`
- Global --> `GLOBAL` --> `ALWAYS`

Reply:
My vote is for `exact`, `fuzzy`, and `always`. ([slack convo](https://elastic.slack.com/archives/C02U50QNEAG/p1690406059257269?thread_ts=1690403731.484959&cid=C02U50QNEAG))